# Request
int32 pin_id
int32 pin_value
---
# No response